sympy/physics/units/dimensions.py

```diff
@@ -270,6 +270,7 @@ def has_integer_powers(self, dim_sys):
 temperature = Dimension("temperature", "T")
 amount_of_substance = Dimension("amount_of_substance")
 luminous_intensity = Dimension("luminous_intensity")
+nucleas_decays = Dimension("nucleas_decays")
```

Comment:
Typo: nucleas -> nucleus

Reply:
Also, isn't the number of nucleus decays dimensionless?

Reply:
It's dimensionless in the standard dimension systems, but one could still define a dimension system where it has its own dimension. The question about how to define dimensions exactly it's still open, technically they should be associated with a `DimensionSystem` object.